Repository/CustomItemRepository.php

```diff
@@ -78,6 +78,21 @@ public function getTableAlias(): string
         return CustomItem::TABLE_ALIAS;
     }
 
+    /**
+     * @throws \Doctrine\ORM\NonUniqueResultException
+     * @throws \Doctrine\ORM\NoResultException
+     */
+    public function getItemCount(int $customObjectId): int
```

Comment:
It looks like this method is used only in the tests. Production code should not be added only to be used by tests.

You can replace this method by this one-liner:

```php
$repository->count(['customObject' => $customObjectId]);
```